app/controllers/prior_authority/steps/add_another_controller.rb

```diff
@@ -0,0 +1,68 @@
+module PriorAuthority
+  module Steps
+    class AddAnotherController < BaseController
```

Comment:
can we put this at the lower level so it can be used for NSM Add Another forms? 

is it worth looking at that for this PR or leave it to review later?

Reply:
Inheritance was making this complicated, so I've moved this into a concern so it can be reused. Until we have a full add-another flow working and signed off in PA I'm hesitant to try to rework NSM to use the same system, but once it's up and running I think it would be great to see if we use this for NSM.